### PR TITLE
#18 updated Region javadoc

### PR DESCRIPTION
**Pull request by darkled**

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks for the pull request, let me find a reviewer..

**Comment by dmarkov** [comment]
@yegor256 it's yours, please review

File: src/main/java/com/jcabi/dynamo/Region.java [hunk 1]
```diff
@@ -49,6 +49,10 @@
  *
  * <pre> Region region = new ReRegion(new Region.Simple(credentials));</pre>
  *
+ * <p>After all operations with the region are finished, it can be optionally
+ * shutdown invoking {@link AmazonDynamoDB#shutdown()} in order to explicitly
+ * release any open resources.
```
Comment (yegor256) [inline src/main/java/com/jcabi/dynamo/Region.java]: optionally? what will happen if I don't do a shutdown? I think we should explain a bit more here...

**Comment by yegor256** [comment]
@darkled see my comment above. The text you added looks good, but I think we need to put a bit more information there. Do our users **have to** shutdown the AWS client or not?

**Comment by darkled** [comment]
@yegor256 some information concerning this question is in the javadoc of `com.amazonaws.services.dynamodbv2.AmazonDynamoDB.shutdown()`:
```java
    /**
     * Shuts down this client object, releasing any resources that might be held
     * open. This is an optional method, and callers are not expected to call
     * it, but can if they want to explicitly release any open resources. Once a
     * client has been shutdown, it should not be used to make any more
     * requests.
     */
    public void shutdown();
```
And in the javadoc of `com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient.shutdown()`:
````java
    /**
     * Shuts down the client, releasing all managed resources. This includes
     * forcibly terminating all pending asynchronous service calls. Clients who
     * wish to give pending asynchronous service calls time to complete should
     * call getExecutorService().shutdown() followed by
     * getExecutorService().awaitTermination() prior to calling this method.
     */
    @Override
    public void shutdown() {
        super.shutdown();
        executorService.shutdownNow();
    }
```
So, the answer is: our users **have to** shutdown **iff** they want forcibly terminate all pending asynchronous service calls.

Please, check the changes in the javadoc once more, they should be more clear now. shouldn't they?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@darkled yes, now it's clear, thanks!

**Comment by dmarkov** [comment]
@yegor256 Thanks so much! Your account was topped up for **16 mins** (transaction ID is `000-d7b7c853`)... +16 added to your rating, current score is: [+4968](http://www.netbout.com/b/35008?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy now pls

**Pull request merged**